annotations.go

```diff
@@ -144,4 +144,5 @@ var defaultAnnotationValues = MapStringW{
 	"timeout-http-keep-alive":   &StringW{Value: "1m"},
 	"whitelist":                 &StringW{Value: ""},
 	"whitelist-with-rate-limit": &StringW{Value: "false"},
+	"server-ssl":                &StringW{Value: "false"},
```

Comment:
except `ingress.class`

all other annotations are in alphabetical order, please move server-ssl to appropriate position

Reply:
Fixed now